.eslintrc.json

```diff
@@ -40,7 +59,8 @@
     "fp",
     "jsdoc",
     "react",
-    "@typescript-eslint"
+    "@typescript-eslint",
+    "disable"
```

Comment:
What does "disable" do?

Reply:
Sorry, this needs to be removed. It was added in order to disable all rules from `ts plugins` for `non-ts files`. But the suggested way was to use `overrides` for that.